extensions/mssql/src/objectManagement/ui/databaseDialog.ts

```diff
@@ -49,10 +65,23 @@ export class DatabaseDialog extends ObjectManagementDialogBase<Database, Databas
 			}
 			this.formContainer.addItems(components);
 		} else {
+			// Options of a boolean dropdown
+			this.booleanOptionsArray = Object.keys(BooleanDropdownOptions);
```

Comment:
Is there a reason you aren't using a checkbox here instead?

Reply:
These options are boolean dropdowns in SSMS and replicating the same here.

Reply:
@erinstellato-ms Any issues with using checkboxes where there's just boolean values? We don't need to replicate the SSMS UI exactly IMO in situations where it doesn't make much sense.

Reply:
The only situation where a dropdown would be preferred is if "blank" or "none" is an option that a user can choose specifically. Although we should try to figure out why/when a user would want to choose that vs explicitly selecting a value.

Reply:
Agree with you! yeah we are displaying just true/false only and no other additonal options. Checking with Erin regarding this question! Thanks

Reply:
@ssreerama @Charles-Gagnon I am good with using a checkbox with the caveat that I'll be interested to see if folks notice and/or complain :)

Reply:
Looks great to me, thanks!

Reply:
> Thanks for the confirmation @erinstellato-ms, i have added screenShots in the description with the checkboxes, please let me know your thoughts on the changes, Thanks.

It looks so different, but it's good :)
One question - I don't see Ledger as an option for your SQL 2022 database.  I believe that should be listed for SQL 2022 (not for SQL 2019 and below though).

Reply:
@erinstellato-ms , Ledger option is there for sql 2022, which is the middle image, it is between the automatic and recovery sections.

Reply:
Ah, I was thinking that was Azure SQL DB.  My bad!  However, that option should also appear for Azure SQL DB.  Is that the left dialog?  (because I don't see it there)

Reply:
No currently we are supporting Onprem and MI right!! Those ss are for local instance>sql22>MI

Reply:
@ssreerama ah, got it.  But we will (eventually) support Azure SQL DB too, right?

Reply:
Yes we do!